src/services/dataFetcher.ts

```
@@ -96,11 +133,7 @@ export class DataFetcher {
       }
 
       // Cache the fresh data
-      storageManager.setTileState<T>(storageKey, {
-        data: data as unknown as T,
-        lastDataRequest: Date.now(),
-        lastDataRequestSuccessful: true,
-      });
+      this.setTileState<T>(storageKey, data as unknown as T, true);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)